rotkehlchen/db/upgrades/v37_v38.py

```diff
@@ -17,6 +17,22 @@ def _add_polygon_pos_location(write_cursor: 'DBCursor') -> None:
     write_cursor.execute('INSERT OR IGNORE INTO location(location, seq) VALUES ("h", 40);')
 
 
+def _add_polygon_pos_nodes(write_cursor: 'DBCursor') -> None:
+    default_polygon_pos_rpc_nodes = [
+        ('polygon etherscan', '', 0, 1, 0.25, 'POLYGON_POS'),
+        ('ankr', 'https://rpc.ankr.com/polygon', 0, 1, 0.15, 'POLYGON_POS'),
+        ('BlockPi', 'https://polygon.blockpi.network/v1/rpc/public', 0, 1, 0.15, 'POLYGON_POS'),
```

Comment:
I took the free nodes from rpclist.com

> Do you have a comparasion of which one is better in terms of speed and privacy?

rpclist.com for some reason is down right now, but as far as I remember all the nodes had relatively same characteristics. In terms of privacy, how would you suggest to check that?

> Also this one offers a full node in the free tier. Have you checked the others?

Afaik defillama offers an archieve node too, but I haven't checked. Does it matter for the upgrade though? Nodes capabilities are determined on the fly anyway

Reply:
In order to check the latency of the nodes I looked at https://chainlist.org/chain/137.

By monitoring it for a bit I can say that nodes' latency jumps up and down, so it's not something really stable. So I would say that it will be okay to have same priority of all the nodes.